jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -78,3 +81,21 @@ opens jdk.internal.misc to
 opens java.lang to
     openj9.criu;
 /*[ENDIF] CRIU_SUPPORT */
+/*[IF OPENJCEPLUS_SUPPORT]*/
+exports jdk.internal.logger to
+    openjceplus;
+/*[IF !CRIU_SUPPORT]*/
+exports jdk.internal.misc to
+    openjceplus;
```

Comment:
This was already exported on line 72

Reply:
@pshipton 
I added the check here, is for, when the CRIU is not supported, then the line 72 will not be added into the module-info.java file. So, I added a check here, if CRIU is not supported, then add the `exports jdk.internal.misc to  openjceplus;` here for OpenJCEPlus.

Reply:
I'm not 100% sure that will work, but we can try if you haven't already.

Reply:
Anyway if we move all this to the extensions then it doesn't matter.

Reply:
Moved all these to the extensions module-info.java.